### PR TITLE
FEATURE: rlc_component without name

### PR DESCRIPTION
**Pull request by gkorompi**

### CODE REVIEW

File: tests/legacy/system/test_edb_modeler.py [hunk 1]
```diff
@@ -466,7 +466,7 @@ def rlc_component_302(self):
         self.local_scratch.copyfolder(source_path_edb, target_path_edb)
         edbapp = Edb(target_path_edb, desktop_version)
         pins = edbapp.components.get_pin_from_component("C31")
-        assert edbapp.components.create_rlc_component([pins[0], pins[1]], r_value=0, component_name="TEST")
+        assert edbapp.siwave.create_rlc_component([pins[0], pins[1]])
```
Comment (SMoraisAnsys) [inline tests/legacy/system/test_edb_modeler.py:469]: Any reason for such change ?

Comment (gkorompi) [reply tests/legacy/system/test_edb_modeler.py:469]: instead of writing a new test i just added this line, which tests the changes.

**Review by SMoraisAnsys — DISMISSED** [review]
LGTM

File: tests/legacy/system/test_edb_modeler.py (hunk 1 above)
Comment (SMoraisAnsys) [reply tests/legacy/system/test_edb_modeler.py:470]: I see that now the two lines are kept so my previous comment does not apply any more.

**Comment by SMoraisAnsys** [comment]
@gkorompi Please avoid using auto merge because I can't accept a review otherwise (my approval would make the PR getting merge). I prefer that either @ring630 or @svandenb-dev also give their opinion before merging something as they are the core maintainers.

**Review by ring630 — APPROVED** [review]
LGTM

**Pull request merged**